themes/generic/src/main/assets/source.st

```diff
@@ -0,0 +1,7 @@
+<div class="source-github">
+$if(page.source)$
+The source code for this page can be found <a href="$page.source$">here</a>.
+$else$
+No associated source code found for this page.
```

Comment:
I would leave this out. The only reason for this would be generated paradox files. For Akka HTTP there will likely be some, in which case the relative path should probably be configurable via the front-matter somehow to for example point to a Scala source file.